managed/src/main/java/com/yugabyte/yw/controllers/CertificateController.java

```diff
@@ -145,9 +167,38 @@ public Result get(UUID customerUUID, String label) {
     }
   }
 
+  public Result delete(UUID certificate) {
+	CertificateInfo cert = CertificateInfo.get(certificate);
+	if (cert == null) {
+		return ApiResponse.error(BAD_REQUEST, "Invalid certificate.");
+	}
+	List<Universe> universes = Universe.getAll();
+	for (Universe universe : universes) {
+		// Try catch block for the universes which does not have certificate attached to it
+		try {
+			Universe universe_obj = Universe.get(universe.universeUUID);
+			UUID certificate_uuid = universe_obj.getUniverseDetails().rootCA;
+			if (certificate_uuid.toString() == cert.uuid.toString()) {
+				return ApiResponse.error(BAD_REQUEST, "The certificate is in use.");
+			}
+		} catch (NullPointerException a) {
```

Comment:
Do not use exceptions for control flow.

Reply:
Same reason as list API exception block.

Reply:
Added a comment in the code as well